tests/gh-actions/install_pipelines.sh

```diff
@@ -7,4 +7,4 @@ kubectl apply -f third-party/metacontroller/base/crd.yaml
 echo "Waiting for crd/compositecontrollers.metacontroller.k8s.io to be available ..."
 kubectl wait --for condition=established --timeout=30s crd/compositecontrollers.metacontroller.k8s.io
 kustomize build env/cert-manager/platform-agnostic-multi-user | kubectl apply -f -
-kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout 180s
\ No newline at end of file
+kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout 300s
```

Comment:
@kimwnasptd heads up that I had to increase the time to allow pods to be ready and pass this condition. When testing locally, it took about ~4-5 mins for all pods to come up for me

Reply:
@annajung let's actually double that, to have some leeway. Let's do 600s

Reply:
Done - I agree that increasing the timeout doesn't hurt since when the condition is all met, it completes.